assets/css/main.css

```diff
@@ -304,8 +304,13 @@ mark {
     max-width: 100%;
   }
 
+  &.wide {
+    max-width: 60%;
+  }
+
   @media (--phone) {
     padding: 20px;
+    max-width: 100% !important;
```

Comment:
I'm almost certain that this is not the best way to force full width on phones; any help appreciated.